applications/cosmoly.md

```diff
@@ -0,0 +1,127 @@
+
+# VeChain General Grant Application Template
+
+## Project Overview
+
+- Project: Vechain integration in Cosmoly
+- Team Name: CosmolyLabs
+- Payment Address: 0xD81F2144996A9A1eB0d9B8260bab25f5485466f0
+
+### Overview
+
+Please provide the following:
+
+- Cosmoly is a app that allows users to interconnect wallets from different blockchains through a single platform. A place where you can exchange, send, receive and save between multiple chains using a single platform with just your wallet.
+- Vechain is a fairly fast and low-cost protocol, but its DeFi ecosystem still needs to be more robust. We want to implement it on the platform so that users can access more tokens and different blockchains, natively using only their wallet.
+- Not only is it another app on vechain, it would bring the use of more tokens, active users and new savings systems (or staking) which would be a perfect opportunity to create new opportunities for developers
+
+### Project Details
+
+Cosmoly is a set of decentralized protocols and services, which operate across different blockchains. Although it can be divided into the following core services:
+
+#### External services
+
+- **Li.Fi Integration:** Providing Liquidity from 28+ DEXs and 15+ Bridges and +22 different blockchains
+- **Onramper:** Providing 17 onramps and 182 local payment methods.
+- **Nexo Earn API**: Off-chain earning services with 5-16% APY (only with some tokens and stable coins)
+
+#### Internal services
+
+- **Bridge between VeChain and Polygon (and 22 blockchains more):** This would allow VeChain to connect (using polygon) to the more than 22 different blockchains of the Li.Fi protocol. Without the need to go through an CEX, registrations or verifications.
+- **Earn protocolo in VeChain:** Earn daily interest using VeUSD, VET and Cosmo-USD (a token collateralized in USDC/USDT).
+- **Creation of COSMO-USD:** A token collateralized in USDC/USDT on the polygon blockchain, which, using the Li.Fi protocol, connects to more stablecoins.
+
+#### Benefits of working with VeChain Services
```

Comment:
Could you provide the transaction stats of your platform? This would greatly benefit the committee in their review of the project.